site/components/PodcastCard/index.tsx

```diff
@@ -19,14 +23,19 @@ export function PodcastCard(props: CardProps) {
         <Text t="body2" className="summary">
           {props.podcast.summary}
         </Text>
+        {props.podcast.rssId && (
+          <A href={`${urls.podcast}/${props.podcast.rssId}/?listen-on=true`}>
```

Comment:
can you make the whole card clickable actually? Same as how the blog post cards work

Reply:
Done